R/archive.R

```diff
@@ -297,7 +300,8 @@ new_epi_archive <- function(
     compactify,
     clobberable_versions_start,
     versions_end,
-    compactify_tol = .Machine$double.eps^0.5) {
+    compactify_tol = .Machine$double.eps^0.5,
+    subclass = character()) {
```

Comment:
What is the intended use case for this that can't be solved with `ea <- new_epi_archive(...); class(ea) <- c(subclass, class(ea))`? Also the term "canonical class" was vague to me, had to read the source to understand. Would be nice to document the intent in the function docstring.

Reply:
I'm going to remove these as "not really necessary".